src/bentoml/_internal/marshal/dispatcher.py

```diff
@@ -190,13 +190,15 @@ async def train_optimizer(
         num_reqs_to_train: int,
         batch_size: int,
```

Comment:
You need to correct the argument too!

Reply:
But we are assigning this batch_size below right? where

Reply:
Why are you doing that? Just rename the argument, this is just less clear what's happening imo.